doc/api/perf_hooks.md

```diff
@@ -37,6 +37,14 @@ added: REPLACEME
 Remove all performance entry objects with `entryType` equal to `name` from the
 Performance Timeline.
 
+### performance.clearGC()
```

Comment:
Sorry for the nit, it seems this should go after `performance.clearFunctions()` ABC-wise)

Reply:
Thanks for the suggestion, I moved it below `performance.clearFunctions()`.